modules/cli-options/src/main/scala/scala/cli/commands/DependencyUpdateOptions.scala

```diff
@@ -0,0 +1,20 @@
+package scala.cli.commands
+
+import caseapp._
+import caseapp.core.help.Help
+
+// format: off
+@HelpMessage("Update dependencies in project")
+final case class DependencyUpdateOptions(
+  @Recurse
+    shared: SharedOptions = SharedOptions(),
```

Comment:
Maybe we could only copy some fields of `SharedOptions` here. Some options are going to be unused for sure here (like the `SharedCompilationServerOptions` ones).

Reply:
But if there's too many of them, maybe don't bother for now…

Reply:
I think this was the idea of SharedOption - so all/most commands has but maybe we can partition it even further to e.g. SharedOption and SharedBuildOptions. For sure, it is not a part of this PR